rocketchat/templates/microservices-stream-hub-deployment.yaml

```
@@ -0,0 +1,79 @@
+{{- if .Values.microservices.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-stream-hub
+  labels:
+    app.kubernetes.io/name: {{ include "rocketchat.name" . }}
+    helm.sh/chart: {{ include "rocketchat.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    {{- if .Values.deploymentLabels }}
+      {{- toYaml .Values.deploymentLabels | nindent 4 }}
+    {{- end }}
+spec:
+  replicas: {{ .Values.microservices.streamHub.replicas }}
+  revisionHistoryLimit: 0
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ .Release.Name }}-stream-hub
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
```

on former line of thinking will having 2 temporarily cause issues?

---

it will not cause issues, it could duplicate db events while both are up..

I think this might be better than losing events though 🤔

---

In this case I would suggest: 
```suggestion
      maxSurge: 1
      maxUnavailable: 0
```